ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/UserManagementServiceIT.scala

```diff
@@ -298,6 +299,129 @@ final class UserManagementServiceIT extends LedgerTestSuite {
     }
   })
 
+  userManagementTest(
```

Comment:
Good point. 
I'm going to add such test after I add a user management cleanup step after each test case (draft PR: https://github.com/digital-asset/daml/pull/12701), as adding it now would be more cumbersome because any other test case can add any number of users without deleting them afterwards

Reply:
@pbatko-da please create a JIRA ticket and then close this comment thread

Reply:
https://digitalasset.atlassian.net/browse/DPP-889